### PR TITLE
Exchange Server Load Balancing mechanism

### PR DESCRIPTION
**Pull request by lalalapotter**
Fixes  ISSUE#319

Changes:

1. Add exchange server load balancing mechanism.

### CODE REVIEW

**Comment by lalalapotter** [comment]
@jiahaoc1993 @LaynePeng Please check, thx! :)

**Comment by LaynePeng** [comment]
> @jiahaoc1993 @LaynePeng Please check, thx! :)

Got it, we will comment in the PR if any question.

File: proposals/Exchange Server Load Balance proposal.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
++ Solve the single point of failure of the exchange server
++ Improve the performance of running multiple tasks when using a single exchange server
+
+## Non-Goals
++ Solve the single point of failure of rollsite and its performance problems in each party.
+
+## UI or API
+Helm Charts `yaml` file
+
+~~~yaml
```
Comment (LaynePeng) [inline proposals/Exchange Server Load Balance proposal.md:18]: Can it be like this:
```
...
module:
  - exchange
 
exchange:
  ip: xxx.xxx.xxx.xxx
  port: 9360
  fedlist:
     - name: party_1
       ip: xxx.xxx.xxx.xxx
       port: xxxx
    - name: party_2
       ip: xxx.xxx.xxx.xxx
       port: xxxx
```

In this case, the exchange just like a normal module, can be add to any FATE-cluster or standalone deployment. 

Alternatively, we can make exchange the same level of FATE or FATE_Serving, which means 
```
...
chartName: Exchange
chartVersion:  v1.5.0
```

Any comments? @jiahaoc1993 @owlet42 . I prefer the first option.

Comment (lalalapotter) [reply proposals/Exchange Server Load Balance proposal.md:18]: I think the first approach is more natural and easy to understand

**Comment by lalalapotter** [comment]
@LaynePeng @jiahaoc1993 Please check the update, thx.

**Review by LaynePeng — APPROVED** [review]
LGTM

**Pull request merged**